### DISCUSSION TITLE
Updates

### DISCUSSION THREAD

**Discussion by joshkemmer** [ixkaito/bathe#33] [Q&A] [answered]
Do you push theme updates to the WordPress interface?

My thought is to create a child theme for Bathe so I can keep up with updates on the parent theme. It would be nice if I can click the button in the WordPress admin, otherwise, I can use something like Git Subtree.

Thanks!

**Comment by ixkaito** [comment, marked as answer]
@joshkemmer Thanks for the comment. In my opinion, it would be better not to publish Bathe to the WordPress theme directory, since Bathe is a starter theme like [Underscores](https://underscores.me/). It is assumed to be developed. You can also easily install the theme by uploading a ZIP file: [How to Install WordPress Plugins and Themes from GitHub](https://www.wpbeginner.com/beginners-guide/how-to-install-wordpress-plugins-and-themes-from-github/ ).
